steps/run/step.py

```diff
@@ -0,0 +1,27 @@
+import logging
+from relay_sdk import Interface, Dynamic as D
+
+relay = Interface()
+
+name = relay.get(D.name)
+
+params = None
+params_array = None
+
+# if `parameters` is defined, `parameters_array` is ignored
+try:
+  params = relay.get(D.parameters)
+except:
+  try:
+    params_array = relay.get(D.parameters_array)
+  except:
+    params = {}
+
+if params_array:
+  logging.debug(f'Iterating workflow {name} over params array {params_array}')
+  #TODO some throttling or limitation
+  for params in params_array:
+    relay.workflows.run(name, params)
+else:
+    logging.debug(f'Running workflow {name} with params {params}')
+    relay.workflows.run(name, params)
```

Comment:
I don't see the decorators for the workflow run link being set like they are in the shell script. Are these being removed?

Reply:
I don't think it was intentional to remove the decorators, so we should figure out how to get them back.

Reply:
Agreed! I don't want to see that feature get forgotten about. I'm still waiting for the frontend side of it :D.